### PR TITLE
AAE-23375 Fixed the issue to import a project with multiInstance with…

### PR DESCRIPTION
**Pull request by AkhileshPamidimarthi**
… null as cardinality

**What kind of change does this PR introduce?**

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

## Description

- Issue Link: https://hyland.atlassian.net/browse/AAE-23375

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

### CODE REVIEW

File: activiti-core/activiti-bpmn-converter/src/main/java/org/activiti/bpmn/converter/export/MultiInstanceExport.java [hunk 1]
```diff
@@ -59,8 +59,11 @@ public static void writeMultiInstance(Activity activity, XMLStreamWriter xtw) th
               xtw.writeCharacters(multiInstanceObject.getCompletionCondition());
               xtw.writeEndElement();
           }
-          xtw.writeEndElement();
       }
+      else{
```
Comment (jesty) [inline activiti-core/activiti-bpmn-converter/src/main/java/org/activiti/bpmn/converter/export/MultiInstanceExport.java]: this else statement containse the first line of the if above, so proabably we can just move out of the if. Can you also add a test? Thanks!

**Review by jesty — CHANGES_REQUESTED** [review]

File: activiti-core/activiti-bpmn-converter/src/test/java/org/activiti/bpmn/converter/export/MultiInstanceExportTest.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+        Activity activity = mock(Activity.class);
+        MultiInstanceLoopCharacteristics multiInstance = new MultiInstanceLoopCharacteristics();
+        when(activity.getLoopCharacteristics()).thenReturn(multiInstance);
+
+
+        MultiInstanceExport.writeMultiInstance(activity, xtw);
+
+
+        verify(xtw).writeStartElement("multiInstanceLoopCharacteristics");
```
Comment (jesty) [inline activiti-core/activiti-bpmn-converter/src/test/java/org/activiti/bpmn/converter/export/MultiInstanceExportTest.java]: can we verify the generated XML?

File: activiti-core/activiti-bpmn-converter/src/test/java/org/activiti/bpmn/converter/export/MultiInstanceExportTest.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+import java.io.StringWriter;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class MultiInstanceExportTest {
```
Comment (jesty) [inline activiti-core/activiti-bpmn-converter/src/test/java/org/activiti/bpmn/converter/export/MultiInstanceExportTest.java:37]: should we test also when the tag it's empty?  I'm expecting a test with an assert like:
`assertTrue(generatedXml.contains("<multiInstanceLoopCharacteristics />"));`
Or am I wrong? Thanks

File: activiti-core/activiti-bpmn-model/src/main/java/org/activiti/bpmn/model/MultiInstanceLoopCharacteristics.java [hunk 4]
```diff
@@ -22,7 +22,7 @@ public class MultiInstanceLoopCharacteristics extends BaseElement {
   private String completionCondition;
   private String elementVariable;
   private String elementIndexVariable;
-  private boolean sequential;
+  private Boolean sequential;
```
Comment (balsarori) [inline activiti-core/activiti-bpmn-model/src/main/java/org/activiti/bpmn/model/MultiInstanceLoopCharacteristics.java]: Why change it to a `Boolean`? sequential should be `false` by default. Now there needs to be lots of null checks.

**Review by jesty — APPROVED** [review]
LGTM

**Pull request merged**